Assets/Scripts/UI/Dialogue/DialogueManagerUI.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class DialogueManagerUI : DialogueManager
+{
+    [SerializeField] private TextMeshProUGUI field;
+
+    void Update() {
+        if (!field) field = FindObjectOfType<TextMeshProUGUI>();
```

Comment:
This line here seems like a red flag. If field is never assigned, just grab a random textmesh from anywhere in the scene?

Reply:
Yeah I guess you're right. This was meant to catch instances where we forget to assign the field. It wouldn't catch any textmesh, it would look specifically for the textmeshpro ui version.

It was a 'in the moment' decision, but looking at it now, I don't think it's valid.